README.md

```diff
@@ -1,2 +1,23 @@
 # Claymore
-Claymore project for 17356
+
+### Web Framework
+We've decided to use <b>Express</b> as our web framework, for these reasons:
+
+* Learning curve and developers’ previous experience
+* Community support and overall industry use/adaptation
+* Integration with other third party libraries and API’s
+* Simplicity and ease of use
+
+
+### QA/Helpful Tools We Have Set up 
+
+* <b>Eslint</b> is the linter our group uses. We use it to automatically lint our code through the eslint plugin for vscode.
```

Comment:
How to run eslint? If it's not set up yet then mention that here.

Reply:
We have a .eslint file in the front end folder. Install the vscode eslint plugin (https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) to have it lint the code in real time

Reply:
updated README with the link to the plugin